README.md

```diff
@@ -296,3 +297,333 @@ Feature Addition
+
+### Show your support
+[:star: this repo](https://github.com/s-yadav/react-number-format)
+
+### Migrate v2 to v3
+[Migrate](/MIGRATE.md)
+
+### Updates
+For regular updates follow me on [_syadav](https://twitter.com/_syadav)
+
+#### Major updates
+
+#### v4.0.0
+Breaking Changes
+
+- onValueChange no longer receives event object as second parameter, so if you accessing it, it will break.
+
+Feature Addition
+- Support defaultValue prop.
+- Trigger onValueChange if the value is formatted due to prop change.
+- Allow password as type prop.
+- Support indian (lakh) and chinese (wan) style thousand grouping.
+- Always allow . to be typed as decimal separator, even when decimal separator is defined differently
+
+
+#### v3.0.0
+- onChange no longer gets values object. You need to use onValueChange instead. This is done because formatted value may change on onBlur event. calling onChange on onBlur doesn't feel right.
+- decimalPrecision is changed to decimalScale. Precision is the number of digits in a number. Scale is the number of digits to the right of the decimal point in a number.
+- decimalScale by default will not add 0s to match provided decimalScale value like decimalPrecision. You have to set fixedDecimalScale to true.
+- mask can be now array of string in which case mask at specific index will be mapped with the # of the pattern.
+- Value can be passed as string or number, but if it is passed as string it should be either formatted value or if it is a numeric string, you have to set isNumericString props to true.
+- Added support for numbers in prefix / suffix / pattern.
+- Fixed caret position and formatting issues.
+- Lot of bugs and stability fixes ([See release notes](https://github.com/s-yadav/react-number-format/releases/tag/v3.0.0))
+
+#### v2.0.0
+- Added isAllowed prop to validate custom input and reject input based on it.
+- onChange api been changed. Now it receives [values object](#values-object) as second parameter.
+- decimalSeparator no longer support boolean values
+- thousandSeparator accepts only true as boolean (which defaults to ,) or thousandSeparator string
+- decimalPrecision only accepts number now
+- Value can be passed as string or number but if it is passed as string you should maintain the same decimal separator on the string what you provided as decimalSeparator prop.
+- Added back the type prop for the input type attribute (Only text or tel is supported)
+- Enforce cursor to be between prefix and suffix in focus, click or arrow navigation.
+- Lot of bugs and stability fixes ([See release notes](https://github.com/s-yadav/react-number-format/releases))
+
+
+### Development
+- Download the zip
+- `npm install`
+- `npm start` to run example server
+- `npm run test` to test changes
+- `npm run bundle` to bundle files
+
+#### Testing
+Test cases are written in jasmine and run by karma
+Test file : /test/test_input.js
+To run test : `npm run test`
+
+## Contributors
+* [Sudhanshu Yadav](https://github.com/s-yadav)
```

Comment:
Instead of manual adding all contributors. Can we have some tool to do this? So it can be auto-generated before all releases.

Reply:
I will do some research on this & let you know if I find something.